temporian/implementation/numpy/data/tests/test_df_to_event.py

```diff
@@ -61,21 +63,45 @@ def test_correct(self) -> None:
     def test_string_column(self) -> None:
         df = pd.DataFrame(
             [
-                [666964, 1.0, "740.0"],
-                [666964, 2.0, "508.0"],
-                [574016, 3.0, "573.0"],
+                [666964, 1.0, "740"],
+                [666964, 2.0, np.nan],
```

Comment:
Following on today's discussion. Can we have a nan in a string array? If so, is that efficient?